HarpGuru/HarpFace/HarpFaceFragment/components/index.tsx

```diff
@@ -9,8 +10,9 @@ import { getHarpRows } from '../../HarpRows'
 export const HarpFaceFragment = (
   props: HarpFaceFragmentProps
 ): React.ReactElement => {
-  const harpRows = getHarpRows(props)
-  const styles = getStyles(props)
+  const [activeHarpStrata, setActiveHarpStrata] = useGlobal('activeHarpStrata')
+  const harpRows = getHarpRows({ ...props, activeHarpStrata })
```

Comment:
These two lines are a little messy, but again, they are so likely to change during the overhaul which will happen after the test mode has been added and reactn is bedded in that it doesn't seem like a good use of time to fix them now. As before, these need recording on an issue to make sure they get done.

Reply:
As above; https://github.com/js-jslog/harpnative/issues/45 created